beacon_node/client/src/broadcast_lightclient_updates.rs

```diff
@@ -0,0 +1,30 @@
+use beacon_chain::{BeaconChain, BeaconChainTypes, LightclientProducerEvent};
+use slog::{error, Logger};
+use tokio::sync::mpsc::Receiver;
+
+// Each LightclientProducerEvent is ~200 bytes. With the lightclient server producing only recent
+// updates it is okay to drop some events in case of overloading. In normal network conditions
+// there's one event emitted per block at most every 12 seconds, while consuming the event should
+// take a few miliseconds. 32 is a small enough arbitrary number.
```

Comment:
small typo:

```suggestion
// Each `LightclientProducerEvent` is ~200 bytes. With the lightclient server producing only recent
// updates it is okay to drop some events in case of overloading. In normal network conditions
// there's one event emitted per block at most every 12 seconds, while consuming the event should
// take a few milliseconds. 32 is a small enough arbitrary number.
```